src/Configuration.txt

```diff
@@ -0,0 +1,12 @@
+authenticationType,JWT
+merchantID,testrest
+merchantsecretKey,yBJxy6LjM2TmcPGu+GaJrHtkke25fPpUX+UY6/L/1tE=

```

Comment:
Can we somehow tag the different set of configuration required for different set of auth mechanism.
like, for JWT you require keyFilename which is not required if someone is using httpSignature.